plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeSystemTables.java

```diff
@@ -103,4 +103,85 @@ public void testPropertiesTable()
             assertUpdate("DROP TABLE IF EXISTS " + tableName);
         }
     }
+
+    @Test
+    public void testPartitionsTable()
+    {
+        String tableName = "test_simple_partitions_table";
```

Comment:
All 3 tests have `_simple_` in the table name. It doesn't provide any benefits unless we have complex tests. I would remove them.